src/sage/topology/cubical_complex.py

```diff
@@ -719,7 +718,7 @@ def _repr_(self):
             sage: C1._repr_()
             '[1,1] x [2,3] x [4,5]'
         """
-        s = ("[%s,%s]" % (str(x), str(y)) for x, y in self.__tuple)
+        s = ("[{},{}]".format(str(x), str(y)) for x, y in self.__tuple)
```

Comment:
Any reason to not use f-strings here? Ruff should be able to do this automatically (UP032).

Reply:
I have deliberately skipped UP032 for the moment.

Reply:
Okay, it's already an improvement anyway.